### PR TITLE
Task 195 implement steps for ui element element

### PR DESCRIPTION
**Pull request by ChepelVE**
add to JDISteps:
        @Given("^I open \"(.*?)\"$")
        @Then("^I'm on \"(.*?)\"$")
	@When("^Click on \"([^\"]*)\"$")
	@When("^Send keys \"([^\"]*)\" to \"([^\"]*)\"$")	
	@When("^Check \"([^\"]*)\"$")	
	@When("^Uncheck \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" label text equals to \"([^\"]*)\"$")
	@Then("^\"([^\"]*)\" label text contains \"([^\"]*)\"$")	
	@When("^Clear \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" value text equals to \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" value text contains \"([^\"]*)\"$")	
	@Then("\"([^\"]*)\" is selected$")	
	@Then("^\"([^\"]*)\" is deselected$")	
	@When("^jsClick on \"([^\"]*)\"$")	
	@When("^Input \"([^\"]*)\" in \"([^\"]*)\"$")	
	@When("^Select \"([^\"]*)\" field from \"([^\"]*)\"$")	
	@When("^Select \"([^\"]*)\" fields from \"([^\"]*)\"$")	
	@When("^Submit \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" placeholder equals to \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" placeholder contains \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" value equals to \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" value contains \"([^\"]*)\"$")	
	@When("^Focus on \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" inner text equals to \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" inner text contains \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" text equals to \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" text contains to \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" attribute \"([^\"]*)\" equals to \"([^\"]*)\"$")	
	@Then("^\"([^\"]*)\" attribute \"([^\"]*)\" contains \"([^\"]*)\"$")	
	@Then("\"([^\"]*)\" is enabled$")	
	@Then("^\"([^\"]*)\" is disabled$")	
	@Then("^\"([^\"]*)\" is displayed$")	
	@Then("^\"([^\"]*)\" is hidden$")	
	@When("^Set text \"([^\"]*)\" in \"([^\"]*)\"$")	
	@When("^Double click on \"([^\"]*)\"$")	
	@When("^Right click on \"([^\"]*)\"$")	
	@When("^Highlight \"([^\"]*)\"$")	
	@When("^Show \"([^\"]*)\"$")	
	@When("^Set \"([^\"]*)\" attribute \"([^\"]*)\" with vlaue \"([^\"]*)\" element$")

### CODE REVIEW

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java [hunk 1]
```diff
@@ -1,35 +1,256 @@
...
+    
+    @Given("^Refresh webpage$")
+    public void refreshWebpage() {
+        WebPage.refresh();
+    }
+   
     @Then("^I'm on \"(.*?)\"$")
     public void iMOn(String pageName) {
         WebPage page = PAGES.get(pageName);
         page.checkOpened();
```
Comment (RomanIovlev) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java]: > shouldBeOpened

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java [hunk 2]
```diff
@@ -1,35 +1,256 @@
...
+	@When("^Uncheck \"([^\"]*)\"$")
+	public void uncheck(String name) {
+		BaseUIElement el = getBaseUI(name);
+		el.uncheck();
+	}
+	
+	@Then("^\"([^\"]*)\" label text equals to \"([^\"]*)\"$")
+	public void labelTextEquals(String name, String value) {
+		BaseUIElement el = getBaseUI(name);
+		assertEquals(value, el.labelText());
```
Comment (RomanIovlev) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java]: name.is().labelText(equalsTo(value))

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java [hunk 3]
```diff
@@ -1,35 +1,256 @@
...
+		assertTrue(el.isSelected());
+	}
+	
+	@Then("^\"([^\"]*)\" is deselected$")
+	public void isDeselected(String name) {
+		BaseUIElement el = getBaseUI(name);
+		assertTrue(el.isDeselected());
+	}
+	
+	@When("^jsClick on \"([^\"]*)\"$")
```
Comment (RomanIovlev) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java]: Click with JavaScript on

File: jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java [hunk 4]
```diff
@@ -21,14 +9,27 @@
...
 @RunWith(Cucumber.class)
 @CucumberOptions(
         features = "classpath:features",
         glue = {"com.epam.jdi.bdd", "cucmberTests"}
 )
 public class Runner extends AbstractTestNGCucumberTests {
     @BeforeClass
-    public static void setUp() {
+    public static void setUp() {    
         logger.setLogLevel(INFO);
```
Comment (RomanIovlev) [inline jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java]: SMART_SEARCH_LOCATORS.add("[ui=%s]");

File: jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java [hunk 5]
```diff
@@ -21,14 +9,27 @@
...
         initElements(StaticSite.class);
         homePage.open();
```
Comment (RomanIovlev) [inline jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java]: remove checkOpened

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java (hunk 1 above)
Comment (ChepelVE) [reply jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java]: done

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java (hunk 2 above)
Comment (ChepelVE) [reply jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java]: done

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java (hunk 3 above)
Comment (ChepelVE) [reply jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java]: done

File: jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java (hunk 4 above)
Comment (ChepelVE) [reply jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java]: done

File: jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java (hunk 5 above)
Comment (ChepelVE) [reply jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java]: done

**Pull request merged**